### DISCUSSION TITLE
What's the best way to clear the tree and start fresh?

### DISCUSSION THREAD

**Discussion by 1mike12** [infinitered/ignite#1798] [Q&A] [answered]
Sometimes I'm changing things about the model definitions and the old state is stale and causing errors and I want to start the app from scratch.

Is there something I can do to flush the old tree?

**Comment by jamonholmgren** [comment, marked as answer]
@1mike12 Good question! Easiest way is to do this in your RootStore:

```ts
// RootStore here
.actions((rootStore) => ({
  reset() {
    applySnapshot(rootStore, {})
  }
})
```
